### ISSUE TITLE
Tensorflow release 2.2.0 not found via `pip install`

### ISSUE THREAD

**Issue by hohl** [tensorflow/tensorflow#39265]
**tldr;** pip does not find the new release 2.2.0 published yesterday

**System information**
- Have I written custom code (as opposed to using a stock example script provided in TensorFlow): Not Applicable
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Ubuntu Server 20.04
- TensorFlow installed from (source or binary): binary
- TensorFlow version (use command below): Not Applicable
- Python version: 3.8.2
- PIP version: 20.1
- CUDA/cuDNN version: Not Applicable (10.2)
- GPU model and memory: Not Applicable (RTX 2080 Ti)

**Describe the current behavior**
From the change log and the Github releases I noticed that Tensorflow 2.2.0 was released and that I could upgrade the Tensorflow version currently used in my project (2.2.0rc4). Sadly, `pip install` failed to install the update as it was unable to find release 2.2.0. (Clearing the cache of pip didn't help.) The PyPi website also says latest version is the rc4, not the [2.2.0 release](https://pypi.org/project/tensorflow/#history).

**Describe the expected behavior**
Well, I would like to be able to install Tensorflow 2.2.0 over `pip`.

**Standalone code to reproduce the issue**
`pip install tensorflow==2.2.0`

**Other info / logs**
```
ERROR: Could not find a version that satisfies the requirement tensorflow==2.2.0  (from versions: 2.2.0rc1, 2.2.0rc2, 2.2.0rc3, 2.2.0rc4)
ERROR: No matching distribution found for tensorflow==2.2.0 (from -r projects/hohl.thesis/requirements.txt (line 3))
```

**Comment by mihaimaruseac** [comment]
It seems the pips didn't upload yet. Looking into it

**Comment by goldiegadde** [comment]
@hohl thanks for reporting the issue, the pips are being uploaded now. Sorry for the confusion, we are fixing the automation on our end.

**Comment by ahtik** [comment]
@hohl In addition to the installation issue, you might face a CUDA version compatibility issue. TF 2.2 is built to work with CUDA 10.1. CUDA 10.2 support requires a manual build from source.

But you can add CUDA 10.1 in addition to the existing 10.2 using offline install and keeping the latst driver and adjust the paths for the tensorflow python env, so it's quite easily doable. https://twitter.com/ahtik/status/1238079762758807554 has a few hints, maybe helps.

**Comment by mihaimaruseac** [comment]
No need to build from source, can also be done by symlinking 10.2 to 10.1.

**Comment by ahtik** [comment]
@mihaimaruseac Thanks! Is this the proper [symlinking procedure](https://github.com/tensorflow/profiler/blob/master/docs/profile_multi_gpu.md#linux-setup) to use for CUDA 10.2 with the public release wheels?

Any ideas if something similar is possible on Windows? I couldn't figure it out by myself without a build from source.

**Comment by hohl** [comment]
Thanks for fixing that so quick. Release 2.2.0 was now found through `pip install`.

**Comment by richieakka** [comment]
(venv) C:\>pip install tensorflow==2.2.0
ERROR: Could not find a version that satisfies the requirement tensorflow==2.2.0 (from versions: none)
ERROR: No matching distribution found for tensorflow==2.2.0

still getting error. 

(venv) C:\>python --version
Python 3.8.0

(venv) C:\>pip --version
pip 20.1.1 from c:\venv\lib\site-packages\pip (python 3.8)

**Comment by kclegg** [comment]
Having same exact issue as @richieakka. Are there any working url's to install any of [these](https://pypi.org/project/tensorflow/2.2.0/#modal-close) wheels directly ?

**Comment by clockzhong** [comment]
Today, I've re-installed the my whole Linux environment for tensorflow2.x because of the bug in 
https://github.com/tensorflow/tensorflow/issues/40278
But after I installed the GPU driver, I found I still couldn't install tensorflow 2.20. I've tried many many commands: 
```
 python3 -m pip install tensorflow==2.2.0
Collecting tensorflow==2.2.0
  Cache entry deserialization failed, entry ignored
  Could not find a version that satisfies the requirement tensorflow==2.2.0 (from versions: 0.12.1, 1.0.0, 1.0.1, 1.1.0rc0, 1.1.0rc1, 1.1.0rc2, 1.1.0, 1.2.0rc0, 1.2.0rc1, 1.2.0rc2, 1.2.0, 1.2.1, 1.3.0rc0, 1.3.0rc1, 1.3.0rc2, 1.3.0, 1.4.0rc0, 1.4.0rc1, 1.4.0, 1.4.1, 1.5.0rc0, 1.5.0rc1, 1.5.0, 1.5.1, 1.6.0rc0, 1.6.0rc1, 1.6.0, 1.7.0rc0, 1.7.0rc1, 1.7.0, 1.7.1, 1.8.0rc0, 1.8.0rc1, 1.8.0, 1.9.0rc0, 1.9.0rc1, 1.9.0rc2, 1.9.0, 1.10.0rc0, 1.10.0rc1, 1.10.0, 1.10.1, 1.11.0rc0, 1.11.0rc1, 1.11.0rc2, 1.11.0, 1.12.0rc0, 1.12.0rc1, 1.12.0rc2, 1.12.0, 1.12.2, 1.12.3, 1.13.0rc0, 1.13.0rc1, 1.13.0rc2, 1.13.1, 1.13.2, 1.14.0rc0, 1.14.0rc1, 1.14.0, 2.0.0a0, 2.0.0b0, 2.0.0b1)
No matching distribution found for tensorflow==2.2.0

pip3 install tensorflow==2.2.0
Collecting tensorflow==2.2.0
  Cache entry deserialization failed, entry ignored
  Could not find a version that satisfies the requirement tensorflow==2.2.0 (from versions: 0.12.1, 1.0.0, 1.0.1, 1.1.0rc0, 1.1.0rc1, 1.1.0rc2, 1.1.0, 1.2.0rc0, 1.2.0rc1, 1.2.0rc2, 1.2.0, 1.2.1, 1.3.0rc0, 1.3.0rc1, 1.3.0rc2, 1.3.0, 1.4.0rc0, 1.4.0rc1, 1.4.0, 1.4.1, 1.5.0rc0, 1.5.0rc1, 1.5.0, 1.5.1, 1.6.0rc0, 1.6.0rc1, 1.6.0, 1.7.0rc0, 1.7.0rc1, 1.7.0, 1.7.1, 1.8.0rc0, 1.8.0rc1, 1.8.0, 1.9.0rc0, 1.9.0rc1, 1.9.0rc2, 1.9.0, 1.10.0rc0, 1.10.0rc1, 1.10.0, 1.10.1, 1.11.0rc0, 1.11.0rc1, 1.11.0rc2, 1.11.0, 1.12.0rc0, 1.12.0rc1, 1.12.0rc2, 1.12.0, 1.12.2, 1.12.3, 1.13.0rc0, 1.13.0rc1, 1.13.0rc2, 1.13.1, 1.13.2, 1.14.0rc0, 1.14.0rc1, 1.14.0, 2.0.0a0, 2.0.0b0, 2.0.0b1)
No matching distribution found for tensorflow==2.2.0


```

Why tensorflow2.2.0 disappeared again in pip3 repos? 
And another question: why tensorflow2.2.0 still use pip to install its package? As I know, for all python3 package we should use pip3 instead of pip, my understanding is right? But according to here:
https://www.tensorflow.org/install/pip
I feel the pip&pip3 usage is very messy. And until now, I couldn't find how to install Tensorflow2.2.0 with GPU support.

**Comment by mihaimaruseac** [comment]
@clockzhong please update `pip` on your container (`python3 -m pip install --upgrade pip`)

**Comment by clockzhong** [comment]
@mihaimaruseac Thanks! Yes, it works now. I was totally confused on the pip&pip3&python&python3.

**Comment by rj1024** [comment]
The issue still persists.

**Comment by mihaimaruseac** [comment]
@rj1024 please update `pip` and make sure your Python is on 64 bits. If after these two steps the issue still persists, please open a new issue, filling in issue template. This one has been solved many times already (by upgrading `pip`)
